<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="led_insertName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>사용자이름</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sendMessage">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>470</y>
     <width>75</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>전송</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ted_message">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>341</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>441</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
